mod/wet4/views/default/object/groupforumtopic.php

```diff
@@ -26,9 +26,13 @@
 	return;
 }
 
-
-	$excerpt = elgg_get_excerpt(gc_explode_translation($topic->description, $lang));
-	$description = gc_explode_translation($topic->description, $lang);
+if($topic->description3){
```

Comment:
oddly enough, not everything was transferred over to json format.
i think that this was just modified from a main branch merge, the descriptions are displayed through a hook..